wp-content/themes/workingnyc/assets/js/global-54bb55b.js

```diff
@@ -1628,14 +1628,14 @@
     label: 'Dark Theme',
     classname: 'default',
     icon: 'lucide-moon',
-    version: '2.0.2.1'
```

Comment:
This change and the change in template-home-page-b5c2b31.js were added when I ran the production script

Reply:
Yes I know, I changed it